applicationset/controllers/applicationset_controller_test.go

```diff
@@ -5889,3 +5838,267 @@ spec:
 		})
 	}
 }
+
+func TestAddStatus(t *testing.T) {
```

Comment:
New tests start here, the rest is just the renaming of the progressive sync fields and change to status map